packages/composer/amazeelabs/silverback_gatsby/src/Plugin/GraphQL/DataProducer/ListEntities.php

```diff
@@ -58,11 +59,12 @@ public function resolve(string $type, ?string $bundle, ?int $offset, ?int $limit
       $metadata->addCacheableDependency($entity);
     }
 
-    if ($access) {
-      $this->checkAccess($entities);
```

Comment:
AFAIK this method is not used anywhere else so we can delete it from `EntityQueryBase`.